_data/projects/AnkiDroid.yml

```
@@ -0,0 +1,14 @@
+name: AnkiDroid
+desc: Your secret trick to achieve superhuman information retention.
+site: https://docs.ankidroid.org/
+tags:
+  - java
+  - android
+  - anki
+  - flashcard
+  - retention
+  - memory
+  - learning
+upforgrabs:
+  name: Good first issue!
```

This is why the automated validation is failing - this is a case-sensitive value we use to query via GraphQL:

```suggestion
  name: Good First Issue!
```

---

thanks